guides/tfvc-to-git.md

```diff
@@ -1,47 +1,220 @@
-# TFVC (Team Foundation Version Control) to Git
-
-**Similar Guides:**  
+<!-- markdownlint-disable MD024 MD031 MD032 -->
+<!-- MD024 - Multiple headings with the same content
+     Disabled for the TFS, GCcode and GitHub sub-sections. No other headings
+     make sense.-->
+<!-- MD031 - Fenced code blocks should be surrounded by blank lines
+     Disabled for rendering and formatting purposes. With blank lines, the
+     numbered lists are broken when rendered by the Architect GitHub Pages 
+     theme. -->
+<!-- MD032 - Lists should be surrounded by blank lines
+     Disabled for rendering and formatting purposes. With blank lines, the
+     numbered lists are broken when rendered by the Architect GitHub Pages 
+     theme. -->
```

Comment:
Any way to have these disabled for the specific parts, rather than the whole file?  
I also think the headers (MD024) is important because that is how links are created. Adding something like a ` - Top Section Name` to each of the duplicates is a method I've used in the past.
MD031 - code blocks are breaking the list count anyways, you might have to do it manually.
MD032 - is about going around a whole list block, so this won't actually break the number list.

Reply:
Resolved in latest push. Fixed markdown and removed all rule suppressions.